indico/modules/events/contributions/controllers/management.py

```diff
@@ -176,7 +176,8 @@ def _process(self):
         inherited_location = self.event.location_data
         inherited_location['inheriting'] = True
         contrib_form_class = make_contribution_form(self.event)
-        form = contrib_form_class(obj=FormDefaults(location_data=inherited_location), event=self.event)
+        form = contrib_form_class(obj=FormDefaults(location_data=inherited_location), event=self.event,
+                                  duration=contribution_settings.get(self.event, 'default_duration'))
```

Comment:
convert the `_default_duration` in `ContributionEntryForm` to a property instead of setting it here; `self.event` should already be available there